test/test_pylint_plugins.py

```diff
@@ -13,7 +13,49 @@ def load_tests(unused_loader, tests, unused_pattern):
     """
     # Have to import this in the function, because the module does
     # initialization on import! ugh.
-    from pylint.testutils import make_tests, LintTestUsingFile, cb_test_gen, linter
+    from pylint.testutils import LintTestUsingFile, linter, get_tests_info
+
+    # Inline functions that was deleted from pylint core
```

Comment:
Yeah, I didn't want to put in the work to try and figure out how their test suite works now.